### PR TITLE
feat(entire-pages): re-applied as `PEmpty`

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
- re-applied as "empty" at entire pages

### CODE REVIEW

File: src/common/modules/navigations/gnb/modules/gnb-recent-favorite/modules/GNBFavorite.vue [hunk 1]
```diff
@@ -314,27 +318,22 @@ export default {
...
+}
+
+/* custom design-system component - p-empty */
+:deep(.p-empty) {
+    text-align: center;
+    padding: 3rem 3.25rem;
+    .text {
+        @apply text-gray-400;
+        font-size: 0.875rem;
+        line-height: 1.5;
```
Comment (song2kim) [inline src/common/modules/navigations/gnb/modules/gnb-recent-favorite/modules/GNBFavorite.vue:330]:
```suggestion
        @apply text-paragraph-md;
```

please refer to [this guide](https://storybook.developer.spaceone.dev/?path=/docs/foundation-styles-typography--font-size)

File: packages/cloudforet/language-pack/en.json [hunk 2]
```diff
@@ -972,7 +972,7 @@
 				"DAY": "Day",
 				"MONTH": "Month",
 				"NETWORKING": "Networking",
-				"NO_SERVICE": "No {service}",
+				"NO_SERVICE": "No Database",
```
Comment (yuda110) [inline packages/cloudforet/language-pack/en.json:975]:
```suggestion
				"NO_SERVICE": "No {service}",
```

This is [custom formatting](https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting) of i18n :)
It's used in this way.
```typescript
$t('COMMON.WIDGETS.ALL_SUMMARY.NO_SERVICE', { service: labelMap[activeTab] })
```

File: src/services/home-dashboard/modules/ResourceMap.vue [hunk 3]
```diff
@@ -537,4 +512,23 @@ export default {
...
+
+    @screen xl {
+        @apply col-span-3;
+        padding-top: 50%;
+    }
+
+    @screen 2xl {
+        @apply col-span-3;
+        padding-top: 50%;
+    }
```
Comment (song2kim) [inline src/services/home-dashboard/modules/ResourceMap.vue:532]: Please check the break point rule~~

**Review by song2kim — APPROVED** [review]
LGTM~ I left small comments :)

File: src/services/home-dashboard/modules/PersonalHealthDashboard.vue [hunk 4]
```diff
@@ -304,4 +323,8 @@ export default {
         }
     }
 }
+:deep(.p-empty) {
```
Comment (yuda110) [inline src/services/home-dashboard/modules/PersonalHealthDashboard.vue:326]: nitpick: please add comment when you custom mirinae component!

```suggestion
}
/* custom design-system component - p-empty */
:deep(.p-empty) {
```

File: src/services/home-dashboard/modules/PersonalHealthDashboard.vue [hunk 5]
```diff
@@ -304,4 +323,8 @@ export default {
...
+    padding-top: 2.5rem;
+    padding-bottom: 1.56rem;
```
Comment (yuda110) [inline src/services/home-dashboard/modules/PersonalHealthDashboard.vue:328]: nitpick: If possible, set the rem unit to 2px :) In this case, padding-bottom should be `1.5rem`
스크린샷 2023-03-03 오전 10 31 57

File: src/services/home-dashboard/modules/ServiceAccounts.vue [hunk 6]
```diff
@@ -26,43 +26,47 @@
...
-        <div class="legends">
-            <template v-if="loading">
-                <div v-for="v in skeletons"
-                     :key="v"
-                     class="flex items-center p-4"
-                >
-                    <p-skeleton width="1.5rem"
-                                height="1.5rem"
-                                class="mr-4 flex-shrink-0"
+                <div>
```
Comment (yuda110) [inline src/services/home-dashboard/modules/ServiceAccounts.vue:29]: quibble: Is this necessary tag? 🧐

File: src/services/home-dashboard/modules/TopProjects.vue [hunk 7]
```diff
@@ -438,4 +418,7 @@ export default {
         }
     }
 }
+:deep(.p-empty) {
```
Comment (yuda110) [inline src/services/home-dashboard/modules/TopProjects.vue:421]: nitpick: 🧐

```suggestion
/* custom design-system component - p-empty */
:deep(.p-empty) {
```

File: src/services/home-dashboard/modules/TrustedAdvisor.vue [hunk 8]
```diff
@@ -6,76 +6,94 @@
...
-                                 class="label-wrapper"
-                            >
-                                <p-i :name="tableData[rowNum].icon"
-                                     width="0.875rem"
-                                     height="0.875rem"
-                                     color="inherit transparent"
-                                />
-                                <span class="text hidden lg:inline-block">{{ tableData[rowNum].label }}</span>
+        <p-data-loader
+            :loading="false"
```
Comment (yuda110) [inline src/services/home-dashboard/modules/TrustedAdvisor.vue:10]: quibble: 🥹???

File: src/services/asset-inventory/cloud-service/modules/CloudServices.vue [hunk 9]
```diff
@@ -31,72 +31,83 @@
...
+                                        class="mr-4"
                             />
-                            <div class="content">
-                                <div class="group-name">
-                                    {{ item.group }}
-                                </div>
-                                <div class="name">
-                                    {{ item.name }}
-                                </div>
+                            <div class="grid grid-cols-1 gap-1 w-full">
```
Comment (yuda110) [inline src/services/asset-inventory/cloud-service/modules/CloudServices.vue:48]: suggestion: If there are two or more styled classes, give them a class name :)

File: src/services/asset-inventory/cloud-service/modules/CloudServices.vue [hunk 10]
```diff
@@ -351,12 +364,8 @@ export default {
...
 .no-data-wrapper {
     @apply flex w-full h-full flex-col justify-center items-center;
-    .text {
-        @apply mt-5 text-center text-primary2;
-        font-weight: bold;
-        font-size: 0.875rem;
-        line-height: 1.6;
-    }
+    padding-top: 4.81rem;
+    padding-bottom: 6.12rem;
```
Comment (yuda110) [inline src/services/asset-inventory/cloud-service/modules/CloudServices.vue:368]: nitpick: 🤓

File: src/services/home-dashboard/modules/PersonalHealthDashboard.vue [hunk 11]
```diff
@@ -1,78 +1,97 @@
...
+            </p>
         </template>
-        <p-data-table
-            :loading="loading"
-            :fields="fields"
-            :items="data"
-            :bordered="false"
+        <p-data-loader
+            :data="data"
+            :loading="false"
```
Comment (yuda110) [inline src/services/home-dashboard/modules/PersonalHealthDashboard.vue:11]: nitpick:
```suggestion
            :loading="loading"
```

File: src/common/modules/widgets/DailyUpdates.vue [hunk 12]
```diff
@@ -34,23 +35,27 @@
                     </div>
                 </div>
             </div>
-
-            <div v-else-if="data.length === 0 && warningData.length === 0"
-                 class="w-full h-full flex items-center"
+            <p-data-loader
+                v-else
+                :loading="false"
```
Comment (yuda110) [inline src/common/modules/widgets/DailyUpdates.vue:40]: nitpick: 🤓
```suggestion
                :loading="loading"
```

**Review by yuda110 — APPROVED** [review]
Nice work :) 👍

**Pull request merged**